extra-cinnamon/bulky/autobuild/postinst

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+
+case "$1" in
```

Comment:
This file is completely unnecessary. This will be done as part of the `glib` trigger.